doc/DeveloperGuide.md

```diff
@@ -61,6 +61,9 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
 `* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
 `*` | user with many persons in the address book | sort persons by name | locate a person easily
+`*`| user | locate a person by address | find direction to visit his home.
```

Comment:
Your what and so that doesn't really match here. You want address book to provide directions, or search by address?

Reply:
Hi, locate as in using his address to locate him, not search for him in the address book by address. Thanks for the comment.